synth/src/sampler.rs

```diff
@@ -13,18 +12,19 @@ pub(crate) struct Sampler {
     graph: Graph,
 }
 
-impl Sampler {
-    fn sampler_progress_bar(target: u64) -> ProgressBar {
-        let bar = ProgressBar::new(target as u64);
-        let style = ProgressStyle::default_bar()
-            .template("[{elapsed_precise}] {wide_bar} {pos}/{len} generated ({eta} remaining)");
-        bar.set_style(style);
-        bar
-    }
+fn sampler_progress_bar(target: u64) -> ProgressBar {
+    let bar = ProgressBar::new(target as u64);
+    let style = ProgressStyle::default_bar()
+        .template("[{elapsed_precise}] {wide_bar} {pos}/{len} generated ({eta} remaining)");
+    bar.set_style(style);
+    bar
+}
 
+impl Sampler {
     #[cfg(feature = "api")]
-    pub(crate) fn sample(self, collection_name: Option<Name>, target: usize) -> Result<Value> {
-        self.sample_seeded(collection_name, target, 0)
+    pub(crate) fn sample(self, collection_name: Option<Name>, target: usize) -> Result<JsonValue> {
+        // self.sample_seeded(collection_name, target, 0)
```

Comment:
Please remove this comment, or, even better, the whole impl (or barring that the whole api feature).

Reply:
Good catch!